aurora-postgres/parameter-group/main.tf

```diff
@@ -0,0 +1,31 @@
+locals {
+  version_components = split(".", var.engine_version)
+  postgres_family = join(
+    "",
+    [
+      "postgres",
```

Comment:
![Screenshot 2024-04-02 at 4 03 42 PM](https://github.com/thoughtbot/terraform-aws-databases/assets/82670787/395343db-220a-4a14-afc4-0cfa5947563a)

Similar comment here, if we want this for aurora maybe we're leaning in and going with an aurora family? 

Very open to debate but it seems inline with a new module.